src/concordances.cpp

```diff
@@ -0,0 +1,27 @@
+#include <Rcpp.h>
+using namespace Rcpp;
+
+int sign(double x){
+  return x == 0 ? 0 : x > 0 ? 1 : -1;
+}
+
+// Defined in Hollander, Wolfe & Chicken, Nonparametric Statistical Methods, 3ed., eq. (8.5), p. 394
+int Q(double a, double b, double c, double d) {
+  return sign((d-b)*(c-a));
+}
```

Comment:
C++ code in jasp uses tabs for indentation. I realize that this is a bit of a different situation but it might be nice to stick to one standard.

Reply:
Ok, did not know that, changed